Production/test/dict_cff_Summer20UL16.py

```diff
@@ -0,0 +1,145 @@
+# format for dict entries:
```

Comment:
Please remove this file and the other `Production/test/dict_cff_Summer20UL*` files like it. We don't normally add them to the repo.

Reply:
I realized we don't have cff_ dictionary in the framework, but I pushed it for checking by you. 
I will remove them.